### PR TITLE
New script: ReadQRCode

### PR DESCRIPTION
**Pull request by jlevypaloalto**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-4396)

## Description
Create a new script that will extract the text of QR code image.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CommonScripts/ReleaseNotes/1_12_55.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### New: ReadQRCode
+
+- New: Extracts the text from a QR code. (Available from Cortex XSOAR 6.10.0).
```
Comment (JudahSchwartz) [inline Packs/CommonScripts/ReleaseNotes/1_12_55.md:6]: check that this double New language is standard

File: Packs/CommonScripts/Scripts/ReadQRCode/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+Extracts the text from a QR code.
+This script uses the open-source library OpenCV and is restricted to it's capabilities.
+The output of this script includes the output of the script "extractIndicators" run on the text extracted from the QR code.
+
+## Script Data
```
Comment (JudahSchwartz) [inline Packs/CommonScripts/Scripts/ReadQRCode/README.md:5]:
```suggestion

This script uses the open-source library OpenCV to extract text from QR codes.
The output of this script includes the output of the script "extractIndicators" run on the text extracted from the QR code.

## Script Data
```

File: Packs/CommonScripts/Scripts/ReadQRCode/ReadQRCode.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+
+def extract_info_from_qr_code(entry_id: str) -> CommandResults:
+
+    try:
+        filename = demisto.getFilePath(entry_id)['path']
+        text = read_qr_code(filename)
+        if not text:
+            return CommandResults(readable_output='No QR code was found in the image.')
+        indicators = extract_indicators_from_text(text)
```
Comment (JudahSchwartz) [inline Packs/CommonScripts/Scripts/ReadQRCode/ReadQRCode.py:29]: check w Tomer if this really should be done here in the script

File: Packs/CommonScripts/Scripts/ReadQRCode/ReadQRCode.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
+import demistomock as demisto  # noqa
+from CommonServerPython import *  # noqa
+import cv2
+
+
+def read_qr_code(filename: str) -> str:
+
+    detect = cv2.QRCodeDetector()
+    img = cv2.imread(filename)
+    text, *_ = detect.detectAndDecode(img)
```
Comment (JudahSchwartz) [inline Packs/CommonScripts/Scripts/ReadQRCode/ReadQRCode.py:10]: what else does this return that we are ignoring? Lets log it

File: Packs/CommonScripts/ReleaseNotes/1_12_55.md (hunk 1 above)
Comment (jlevypaloalto) [reply Packs/CommonScripts/ReleaseNotes/1_12_55.md:6]: This is how it was generated

File: Packs/CommonScripts/Scripts/ReadQRCode/ReadQRCode.py (hunk 4 above)
Comment (jlevypaloalto) [reply Packs/CommonScripts/Scripts/ReadQRCode/ReadQRCode.py:10]: done

**Review by JudahSchwartz — APPROVED** [review]
🤠

**Pull request merged**